tests/test_modelldcatnofactory.py

```diff
@@ -389,3 +394,51 @@ def test_creates_specialization_property(mocker: MockerFixture) -> None:
     g2 = Graph().parse(data=actual.to_rdf(), format="turtle")
 
     assert_isomorphic(g1, g2)
+
+
+def test_creates_attribute_model_property(mocker: MockerFixture) -> None:
+    """Test that attribute properties are correctly created."""
+    attribute_identifier = "identifier"
+    type = "string"
+    title = {None: "title"}
+    description = {None: "description"}
+    max_occurs = "1"
+    min_occurs = "1"
+
+    mock_component = mocker.MagicMock()
+    mock_component.identifier = attribute_identifier
+    mock_component.title = title
+    mock_component.description = description
+    mock_component.type = type
+    mock_component.format = None
+    mock_component.min_occurs = min_occurs
+    mock_component.max_occurs = max_occurs
+
+    mock_schema = mocker.MagicMock()
+
+    simple_type_identifier = "simple_identifier"
+    simple_type = SimpleType(simple_type_identifier)
+    simple_type.title = {None: type}
+    simple_type.type_definition_reference = TYPE_DEFINITION_REFERENCE.get(type)
+
+    expected = Attribute(attribute_identifier)
+    expected.title = title
+    expected.description = description
+    expected.max_occurs = max_occurs
+    expected.min_occurs = min_occurs
+    expected.has_simple_type = simple_type
+
+    mocker.patch("jsonschematordf.schema.Schema.add_parsed_component")
+    mocker.patch(
+        "jsonschematordf.modelldcatnofactory._create_identifier",
+        side_effect=[attribute_identifier, simple_type_identifier],
+    )
+
+    actual = modelldcatno_factory._create_attribute_property(
+        mock_component, mock_schema
+    )
+
+    g1 = Graph().parse(data=expected.to_rdf(), format="turtle")
```

Comment:
Endre expected til å være ferdig serialisert turtle

Reply:
Hvorfor trengs det?

Reply:
Er ikke hele poenget med biblioteket å produsere rdf?
Vil si det er en veldig mye bedre test om `expected` er faktisk sluttprodukt, as-is føles det veldig halvveis

Reply:
Poenget med biblioteket er å produsere RDF, men ser ikke hvorfor det er nødvendig å jobbe med RDF på såppass "lavt" nivå?

Det er mye mer tungvint å jobbe med strings i RDF format og i min mening vanskeligere å lese og vedlikeholde. Poenget med denne funksjonen og de andre i `modelldcatnofactory` er å ta en `Component` og produsere riktig `modelldcatno` element. Grunnen til at jeg sammenligner grafer her er for å sjekke at alle felt blir satt riktig, men innser at det kanskje er bedre å droppe grafsammenligning og kun sammenligne objekter om vi ikke skal ha en forventet RDF string.

Serialisering til RDF kommer senere, og vil bli testet som del av det øvre API'et i enhets- og integrasjonstester.

Reply:
j. Hvis det er at objektene har fått de korrekte verdiene som testes så er det bedre med sammenligning av selve objektene. Men dette er vel objekter du importerer? Det krever i så fall at de importerte objektene har en bra `__eq__`-metode, hvis det ikke er tilfelle så kan du jo like så greit bruke grafsammenligning.